src/components/data-table/column-sorting/column-sort_spec.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  inject,
+  injectAsync,
+  it,
+  async
+} from '@angular/core/testing';
+import {ComponentFixture, TestComponentBuilder} from "@angular/compiler/testing";
+import {Component, DebugElement, ViewChildren, QueryList, OnInit} from '@angular/core';
+import {CORE_DIRECTIVES} from '@angular/common';
+import {By} from "@angular/platform-browser";
+import {
+  MdDataColumnSortingService,
+  MdDataColumnSortDirective,
+  SortDirection,
+  IColumnSortingModel
+} from './index';
+
+export function main() {
+
+  @Component({
+    selector: 'colsort-test-cmp',
+    providers: [MdDataColumnSortingService],
+    directives: [CORE_DIRECTIVES, MdDataColumnSortDirective],
+    template: `
+    <table>
+      <thead>
+        <tr>
+          <th md-data-column-sort="1">Wine</th>
+          <th md-data-column-sort="2">Cheese</th>
+          <th md-data-column-sort="3">Crackers</th>
+        </tr>
+      </thead>
+    </table>`
+  })
+  class ColsortTestComponent implements OnInit {
+    defaultModel: IColumnSortingModel = {
+      column: '1',
```

Comment:
More clearly what? The point of it being a string is it can be whatever they want, including "1, 2, 3"